pkg/neutronapi/dbsync.go

```diff
@@ -47,7 +49,7 @@ func DbSyncJob(cr *neutronv1beta1.NeutronAPI, scheme *runtime.Scheme) *batchv1.J
 								},
 								{
 									Name:  "DatabaseHost",
-									Value: cr.Spec.DatabaseHostname,
+									Value: cr.Spec.DatabaseInstance,
 								},
```

Comment:
The `DatabaseX` env vars shouldn't be needed here, as they are given to the init container and used there instead.